cornellh4i/ithaca-recovery
# finished DatePicker and TimePicker

### Description
Implemented a Date Picker and Time Picker component that allows users to select a date/time.

### Testing
1. Navigate to [http://localhost:3000/test](http://localhost:3000/test) 
2. Scroll to the bottom of the page, and observe the "DatePicker and TimePicker" section created
3. select the date/time by clicking on the icon to the right

### Notes
- Label is the text next to the picker and can either be a string or JSX.Element (both examples are showcased in the test page)
- if you receive an error, the message is displayed in console.log
- the last prop is in case we want any additional props to be parsed in beyond the core props

related to issue #36
